src/boosting/gbdt.cpp

```diff
@@ -349,6 +353,18 @@ void GBDT::ModelsFromString(const std::string& model_str, int num_used_model) {
   Log::Info("%d models has been loaded\n", models_.size());
 }
 
+std::string GBDT::FeatureImportance(int last_iter) const {
+    size_t* feature_importances = new size_t[max_feature_idx_ + 1]{0};
+    for (int iter = 0; iter < last_iter; ++iter) {
+        for (int split_idx = 0; split_idx < models_.at(iter)->num_leaves() - 1; ++split_idx) {
+            ++feature_importances[models_.at(iter)->split_feature(split_idx)];
+        }
+    }
+    std::string ret = Common::ArrayToString(feature_importances, max_feature_idx_ + 1, ' ');
```

Comment:
can we make this more readable? like feature_1: importance1 \n feature_2:importance2 \n ...